engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableTest.java

```diff
@@ -2703,18 +2707,150 @@ public void testUngroupingAgnostic() {
                 ColumnVectors.ofObject(t1, "Y", String.class).toArray());
     }
 
+    private static class CustomOperationInitializationThreadPool extends EgressInitializationThreadPool {
+        private final AtomicBoolean isUsed = new AtomicBoolean(false);
+
+        CustomOperationInitializationThreadPool() {
```

Comment:
In private inner (static and non-static) classes, I tend to mark all methods public that are intended to be used by the outer class and private for all methods that are internal to the helper.

`private` is stronger than `package private` and is still accessible to the enclosing class(es) -- so private/package-private/public is purely sugar in private inner classes.

Reply:
Will keep this in mind.